assets/src/domain/eventEditor/services/apollo/queries/tickets/useTickets.ts

```diff
@@ -7,7 +7,7 @@ import { getCacheIds } from '@appServices/predicates';
 
 const useTickets = (): Array<Ticket> => {
 	const options = useTicketQueryOptions();
-	const { data } = useCacheQuery<TicketsList>({ ...options, fetchPolicy: 'cache-first' });
+	const { data } = useCacheQuery<TicketsList>(options);
```

Comment:
This caused refetchiing of tickets when deleting or creating a datetime.

Reply:
oh wasn't that added to fix another issue though?